build.gradle

```diff
@@ -69,6 +70,7 @@ allprojects {
             }
         }
         maven { url "https://kotlin.bintray.com/kotlinx" }
+        maven { url kotlinCompilerRepo }
```

Comment:
please document this line and enable it conditionally only for snapshot builds

Reply:
And please elaborate on why teamcity coordinates are required in order to build afu on Kotlin master

Reply:
That's not my commit. Let's create a separate clean review.